charts/portworx/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: portworx
-version: 2.13.0
+version: 2.13.3
 description: A Helm chart for installing Portworx on Kubernetes.
 kubeVersion: ">=1.10.0"
 appVersion: "2.12.2"
```

Comment:
for AWS charts, I have also been updating the appVersion values along with version.
Should that been done here as well